catalyst/core/runner.py

```diff
@@ -184,57 +227,19 @@ def _prepare_for_stage(self, stage: str):
         utils.set_global_seed(self.experiment.initial_seed)
         callbacks = self.experiment.get_callbacks(stage)
 
-        loggers = utils.process_callbacks(
-            OrderedDict(
-                [
-                    (k, v) for k, v in callbacks.items()
-                    if isinstance(v, LoggerCallback)
-                ]
-            )
-        )
-        callbacks = utils.process_callbacks(
-            OrderedDict(
-                [
-                    (k, v) for k, v in callbacks.items()
-                    if not isinstance(v, LoggerCallback)
-                ]
-            )
-        )
-
-        self.state.loggers = loggers
-        self.loggers = loggers
-        self.callbacks = callbacks
+        self.stage_callbacks = self._process_callbacks_for_stage(callbacks)
```

Comment:
I don't think we need new abstraction like `stage_callbacks`, `callbacks ` are good enough to have.
it's also better to use
```
self.callbacks, self.loggers = self._process_callbacks(callbacks)
self.state.loggers = self.loggers
```

also you don't need naming `_process_callbacks_for_stage` because it's already into `_prepare_for_stage ` fn